### PR TITLE
Client Logos

### PR DESCRIPTION
**Pull request by trevorgerhardt**
- Load all clients that are in the folder. 
- Show a logo if one exists. 
- Images added inline to markdown.

### CODE REVIEW

**Comment by trevorgerhardt** [comment]
#### 📝Changed routes:


| [**/clients**](https://conveyal-com-df0kpg0qy.vercel.app/clients) |
|:-:|
|<a href="https://conveyal-com-df0kpg0qy.vercel.app/clients">Screenshot of /clients</a><br /><sup><a href="https://images.weserv.nl?url=https://s.microlink.io/Z28C0Wtpx2L3fr8b_UZI2YURt0o_.png">(view full size)</a> |







Commit 300d58e5cacfe56dc4ace1a3a299d87e48c86188 (https://conveyal-com-df0kpg0qy.vercel.app).

**Review by ansoncfit — COMMENTED** [review]
Does this use anchors to link to the corresponding sections of the client list?

**Comment by trevorgerhardt** [comment]
> Does this use anchors to link to the corresponding sections of the client list?

Now links to the correct client heading 👍

**Review by ansoncfit — COMMENTED** [review]
I pushed a few commits and added a couple comments in-line.

One other observation - the shade of the Indy MPO logo makes it hard to see on the clients page.

File: content/clients/wri.mdx [hunk 1]
```diff
@@ -1,7 +1,10 @@
 ---
 name: World Resources Institute
 id: wri
-image: /img/clients/wri-map.png
 ---
 
-The [World Resources Institute](https://www.wri.org/) Ross Center for Sustainable Cities and its research collaborators have leveraged Conveyal's tools to provide insights into urban accessibility and equity in a range of global cities. Recent projects have analyzed cities in [Turkey](https://thecityfix.com/blog/better-transport-mean-better-access-assessing-infrastructure-investment-izmir-turkey/), [Mexico](https://thecityfix.com/blog/map-month-mobility-health-education-inequality-mexico-city-spatial-problem-mauricio-brito-lorelei-ramirez-reyes-jorge-macias-eric-mackres/), [South Africa](https://www.wri.org/wri-citiesforall/publication/mobility-access-all-expanding-urban-transportation-choices-global-south), and [Kenya](https://doi.org/10.1177/0361198120914620).
+The [World Resources Institute](https://www.wri.org/) Ross Center for Sustainable Cities and its research collaborators have leveraged Conveyal's tools to provide insights into urban accessibility and equity in a range of global cities.
+
+![](/img/clients/wri-map.png)
```
[the remark this answers is not in the document]
Comment (trevorgerhardt) [reply content/clients/wri.mdx]: Fixed 👍

**Review by ansoncfit — APPROVED** [review]

**Pull request merged**